test/webserv/integration/common_response.py

```
@@ -74,6 +91,9 @@ def create_response_header(
     501, CLOSE, not_implemented_file_501_length, TEXT_HTML
 )
 
+created_response_close = response_header_201_close + created_file_201.decode("utf-8")
```

Ruff

Undefined name `created_file_201`

(F821)